src/libsyntax_pos/symbol.rs

```diff
@@ -413,23 +413,30 @@ declare_keywords! {
     (49, Virtual,            "virtual")
     (50, Yield,              "yield")
 
+    // Edition-specific keywords currently in use.
+    (51, Try,                "try") // >= 2018 Edition Only
```

Comment:
Nit: Keywords are not normally moved into the "used" category until the feature is stable (e.g. `macro`/`yield`/`async` are all "reserved for future use").